pkg/bundle/deploy.go

```diff
@@ -2463,6 +2463,23 @@ spec:
                     description: Prometheus namespace that scrap metrics from noobaa
                     type: string
                 type: object
+              bucketLogging:
+                description: BucketLogging sets the configuration for bucket logging
+                properties:
+                  bucketLoggingStorageClass:
+                    description: |-
+                      BucketLoggingStorageClass (otional) specifies the type of storage class that will be used to create a
+                      PVC(Persistent Volume Claim) for guaranteed logging, if enabled.
+                    type: string
+                  loggingType:
+                    description: |-
+                      LoggingType specifies the type of logging for the bucket
+                      There are two types available: guaranteed and non-guaranteed logging
+                      - guaranteed logging ensures that all log entries are recorded without fail
+                      - non-guaranteed logging may not record all log entries
+                      By default, non-guaranteed logging is enabled if no type is specified
```

Comment:
I want to change non-guaranteed to best-effort, as I don't want it to sound like something you shouldn't use.
please change in all the needed places. and about the description, I suggest something like this:
```suggestion
                    description: |-
                      LoggingType specifies the type of logging for the bucket
                      There are two types available: best-effort and guaranteed logging
                      - best-effort(default) - less immune to failures but with better performance
                      - guaranteed - much more reliable but need to provide a storage class that supports RWX PVs
```

Reply:
sure, thanks.